database/Seeders/eggs/source-engine/egg-ark--survival-evolved.json

```diff
@@ -4,29 +4,39 @@
         "version": "PTDL_v1",
         "update_url": null
     },
-    "exported_at": "2021-03-04T06:38:31+02:00",
+    "exported_at": "2021-06-05T15:47:39-04:00",
     "name": "Ark: Survival Evolved",
-    "author": "dev@shepper.fr",
+    "author": "support@pterodactyl.io",
     "description": "As a man or woman stranded, naked, freezing, and starving on the unforgiving shores of a mysterious island called ARK, use your skill and cunning to kill or tame and ride the plethora of leviathan dinosaurs and other primeval creatures roaming the land. Hunt, harvest resources, craft items, grow crops, research technologies, and build shelters to withstand the elements and store valuables, all while teaming up with (or preying upon) hundreds of other players to survive, dominate... and escape! \u2014 Gamepedia: ARK",
     "features": null,
     "images": [
-        "quay.io\/parkervcp\/pterodactyl-images:debian_source"
+        "ghcr.io\/pterodactyl\/games:source"
     ],
-    "startup": "rmv() { echo -e \"stoppping server\"; rcon -t rcon -a 127.0.0.1:${RCON_PORT} -p ${ARK_ADMIN_PASSWORD} -c saveworld && rcon -a 127.0.0.1:${RCON_PORT} -p ${ARK_ADMIN_PASSWORD} -c DoExit; }; trap rmv 15; cd ShooterGame\/Binaries\/Linux && .\/ShooterGameServer {{SERVER_MAP}}?listen?SessionName=\"{{SESSION_NAME}}\"?ServerPassword={{ARK_PASSWORD}}?ServerAdminPassword={{ARK_ADMIN_PASSWORD}}?Port={{SERVER_PORT}}?RCONPort={{RCON_PORT}}?QueryPort={{QUERY_PORT}}?RCONEnabled={{ENABLE_RCON}}$( [ \"$BATTLE_EYE\" == \"0\" ] || printf %s '?-NoBattlEye' ) -server -log & until echo \"waiting for rcon connection...\"; rcon -t rcon -a 127.0.0.1:${RCON_PORT} -p ${ARK_ADMIN_PASSWORD}; do sleep 5; done",
+    "file_denylist": [],
+    "startup": "\"\/home\/container\/ShooterGame\/Binaries\/Linux\/ShooterGameServer {{SERVER_MAP}}?listen?SessionName='{{SESSION_NAME}}'?ServerPassword={{ARK_PASSWORD}}?ServerAdminPassword={{ARK_ADMIN_PASSWORD}}?Port={{PORT}}?MaxPlayers={{SERVER_MAX_PLAYERS}}?RCONPort={{RCON_PORT}}?QueryPort={{QUERY_PORT}}?RCONEnabled={{ENABLE_RCON}} -server -log\"",
     "config": {
         "files": "{}",
-        "startup": "{\r\n    \"done\": \"Waiting commands for 127.0.0.1:\",\r\n    \"userInteraction\": []\r\n}",
+        "startup": "{\r\n    \"done\": \"Setting breakpad minidump AppID = 346110\",\r\n    \"userInteraction\": []\r\n}",
```

Comment:
This is no indication of a successful server startup, it will always show when you try to start the server even if it gets stuck or fails to start. You only see the minidump because you disabled RCON for console output